library/Icinga/File/Ini/IniParser.php

```diff
@@ -260,6 +261,12 @@ public static function parseIniFile($file)
             throw new NotReadableError('Couldn\'t read the file `%s\'', $path);
         }
 
-        return Config::fromArray(parse_ini_string($content, true))->setConfigFile($file);
+        try {
+            $configArray = parse_ini_string($content, true);
+        } catch (ErrorException $e) {
+            throw new ConfigurationError('Couldn\'t parse the INI file `%s\': %s', $path, $e);
```

Comment:
This will print the entire stack trace as output, even if one has stack traces disabled. 

![screen shot 2018-01-18 at 18 52 26](https://user-images.githubusercontent.com/382049/35113297-35dabe1a-fc81-11e7-82d2-3bacd929b4eb.png)


Please use `$e->getMessage()` instead.